src/app/pages/room/room.page.html

```diff
@@ -8,6 +8,16 @@
     </ion-title>
     <ion-buttons slot="end" *ngIf="currentGame$ | async as game">
       <ng-container *ngIf="gameInProgress && userIsInRoom">
+        <ion-button
+          *ngIf="soundService.muted()"
+          tooltip="Toggle Sounds" (click)="toggleSound()">
+          <ion-icon name="volume-mute-outline"></ion-icon>
+        </ion-button>
+        <ion-button
+          *ngIf="!soundService.muted()"
+          tooltip="Toggle Sounds" (click)="toggleSound()">
+          <ion-icon name="volume-high-outline"></ion-icon>
```

Comment:
```suggestion
          <ion-icon [name]="soundIcon"></ion-icon>
```
Maybe just have one button that has a name like this then.... (look for next comment)